test/index.js

```diff
@@ -715,8 +715,8 @@ describe('Nightmare', function () {
       headers['x-nightmare-header'].should.equal('hello world');
     });
 
-    it('should allow web-preferece settings', function*() {
-      nightmare = Nightmare({'web-preferences': {'web-security': false}});
+    it('should allow webPrefereces settings', function*() {
```

Comment:
looks like we made this error before but do you mind correcting this? `webPrefereces => webPreferences`?